torch/_dynamo/variables/builder.py

```diff
@@ -1248,7 +1248,8 @@ def _clone_input(value):
         if isinstance(value, torch.Tensor):
             # tensor subclasses will not be converted to FakeTensors and need to be cloned
             if not (
-                isinstance(value, FakeTensor) or is_fakified_functional_tensor(value)
+                isinstance(value, FakeTensor)
+                or is_fakified_functional_tensor(value, tx.fake_mode)
```

Comment:
I'm confused: since a FunctionalTensor is morally a Tensor subclass, I would expect it should go through this path and go to the clone_input call

Reply:
Oh I see what's going on. The comment is not really meant for AOTDispatch